samples/cxl_host_exerciser/cxl_he_cache_cmd.h

```diff
@@ -94,9 +95,9 @@ class he_cache_cmd : public he_cmd {
         ->default_val("host");
 
     app->add_option("--bias", he_bias_,
-        "host exerciser run on hostmem or fpgamem")
+        "host exerciser run on host or device")
```

Comment:
Change the help to "CXL IP Memory access Bias mode"

Reply:
Fixed